### ISSUE TITLE
Missing buttons for Manage Audiobook section

### ISSUE THREAD

**Issue by saitoh183** [DobyTang/LazyLibrarian#1624]
Ebooks:
![image](https://user-images.githubusercontent.com/1534197/47132562-84bec680-d272-11e8-898f-9a74e5fc9e28.png)

Audiobooks:

![image](https://user-images.githubusercontent.com/1534197/47132580-9ef8a480-d272-11e8-869c-e00a87b521f0.png)

**Comment by philborman** [comment]
Yes that's intentional. The buttons are all to do with ebooks, not audiobooks, at the moment anyway.
Not having the buttons just makes it clearer. Actually that's all except for goodreads sync, could add that one in.

**Comment by philborman** [comment]
Next version of lazylibrarian has separate csv for audio/ebook so there will be buttons

**Comment by saitoh183** [comment]
so until then , i cant manually add audiobooks that havent been snatched by LL or is there a way around that?

**Comment by philborman** [comment]
Drop them into the audiobook folder structure, each book in a separate folder, and run libraryscan from the audiobook page. Lazylibrarian will read the id3 tags and import them. If you have "rename books" ticked in config it will rename/move them into subfolders matching the config options.

**Comment by saitoh183** [comment]
So i added the folder to my /audiobooks and then went under audiobooks and clicked on libraryscan but nothing happened

![image](https://user-images.githubusercontent.com/1534197/47173205-722ca780-d2db-11e8-80f5-7a844bee4b2f.png)

![image](https://user-images.githubusercontent.com/1534197/47173415-00089280-d2dc-11e8-88e3-e01637216bf4.png)

![image](https://user-images.githubusercontent.com/1534197/47173288-adc77180-d2db-11e8-934f-7d7b9d319e6d.png)

**Comment by philborman** [comment]
Would need to see a debug log. Not clear what's going on from the screenshots

**Comment by philborman** [comment]
Just pushed the version with added buttons on the manage page. Should be able to import from the alternate directory too now, but the debug log should show why the libraryscan did not work

**Comment by saitoh183** [comment]
```
18-Oct-2018 14:49:33 - DEBUG   :: AUDIOBOOK_SCAN : librarysync.py:LibraryScan:562 : [/audiobooks] Now scanning subdirectory /M.Y.T.H. Inc. Link - Robert Lynn Asprin
18-Oct-2018 14:49:33 - DEBUG   :: AUDIOBOOK_SCAN : librarysync.py:LibraryScan:629 : file meta [] [Unknown] [Robert Lynn Asprin] [M.Y.T.H. Inc. Link] [148205]
18-Oct-2018 14:49:33 - DEBUG   :: AUDIOBOOK_SCAN : librarysync.py:find_book_in_db:172 : Searching database for [M.Y.T.H. Inc. Link] by [Robert Lynn Asprin]
18-Oct-2018 14:49:33 - DEBUG   :: AUDIOBOOK_SCAN : librarysync.py:find_book_in_db:198 : Exact match [M.Y.T.H. Inc. Link]
18-Oct-2018 14:49:33 - DEBUG   :: AUDIOBOOK_SCAN : bookrename.py:audioProcess:140 : M.Y.T.H. Inc. Link found 1 audiofile
18-Oct-2018 14:49:33 - WARNING :: AUDIOBOOK_SCAN : bookrename.py:audioProcess:146 : M.Y.T.H. Inc. Link: Incorrect number of parts (found 4 from 1)
18-Oct-2018 14:49:33 - DEBUG   :: AUDIOBOOK_SCAN : importer.py:update_totals:389 : Updated totals for [Robert Lynn Asprin]
18-Oct-2018 14:49:33 - INFO    :: AUDIOBOOK_SCAN : librarysync.py:LibraryScan:1026 : 0/0 new/modified AudioBooks found and added to the database
```
![image](https://user-images.githubusercontent.com/1534197/47177201-c3419900-d2e5-11e8-8ff7-ac28a67740fc.png)

I will update to the new version and see

**Comment by philborman** [comment]
Ah - it's a single file audiobook, I haven't got any of those to try. Might still hit the same issue. I think your audio file probably doesn't have any id3 tags, or at least doesn't have all the ones we want, so we try to fake it. Badly. Will push a version that does better checks shortly.

**Comment by saitoh183** [comment]
Cool thanks

**Comment by saitoh183** [comment]
@philborman 

After using the Import Audiobooks under manage, it seems to be keeping original files even if the option isnt checked. i dont know if this is affecting my script which works perfectly when its a auto grab from LL.

![image](https://user-images.githubusercontent.com/1534197/47225209-85974b80-d38b-11e8-8a9f-e130b256d3ef.png)
![image](https://user-images.githubusercontent.com/1534197/47225251-a495dd80-d38b-11e8-82df-961e008f51a1.png)

**Comment by saitoh183** [comment]
> 
> 
> Ah - it's a single file audiobook, I haven't got any of those to try. Might still hit the same issue. I think your audio file probably doesn't have any id3 tags, or at least doesn't have all the ones we want, so we try to fake it. Badly. Will push a version that does better checks shortly.

Also your fix works but this method doesnt launch my custom script and putting that same audiobook in the manual import folder doesnt work.
```
19-Oct-2018 10:21:51 - DEBUG   :: IMPORTALT : postprocess.py:processAlternate:83 : Processing AudioBook directory /downloads/LazyLibrarian/manualimport
19-Oct-2018 10:21:51 - DEBUG   :: IMPORTALT : postprocess.py:processAlternate:83 : Processing AudioBook directory /downloads/LazyLibrarian/manualimport/M.Y.T.H. Inc. Link - Robert Lynn Asprin
19-Oct-2018 10:21:51 - WARNING :: IMPORTALT : postprocess.py:processAlternate:202 : AudioBook /downloads/LazyLibrarian/manualimport/M.Y.T.H. Inc. Link - Robert Lynn Asprin/M.Y.T.H. Inc. Link - Robert Asprin 64 kbps.mp3 has no metadata, unable to import
19-Oct-2018 10:21:51 - WARNING :: IMPORTALT : postprocess.py:processAlternate:146 : No audiobook file found in /downloads/LazyLibrarian/manualimport
```

**Comment by philborman** [comment]
With no metadata we have no way of determining author and title from the manual import folder. Maybe simplest use an id3 tag editor to add metadata to the file. Missing metadata doesn't matter if lazylibrarian downloaded the file as we know what we asked for.

**Comment by saitoh183** [comment]
It was just a snippet of the content. All files were renamed but all originals remained and a playlist.ll was also created. I don't know what is causing my script to fail zipping up the content of the folder but until that problem is fixed, I can't determine if my script is the problem.

**Comment by saitoh183** [comment]
@philborman

But why does it work with library scan when I put the folder in the /audiobook. It was able to rename it and it still didn't have id3 information

**Comment by philborman** [comment]
Will look into this later. Out of action at the moment doing an ubuntu upgrade. 
Looks like it worked from the /audiobook folder because we got
file meta [] [Unknown] [Robert Lynn Asprin] [M.Y.T.H. Inc. Link] [148205]
Searching database for [M.Y.T.H. Inc. Link] by [Robert Lynn Asprin]
but can't see where the file meta came from as it's not id3 tags
M.Y.T.H. Inc. Link - Robert Asprin 64 kbps.mp3 has no metadata
does the file meta message mean we found an .opf file?
